web/src/portico/email_log.js

```diff
@@ -46,4 +47,8 @@ $(() => {
             },
         });
     });
+    $(".open-forward-email-modal").on("click", (e) => {
+        e.preventDefault();
+        overlays.open_modal("forward_email_modal");
+    });
```

Comment:
This one I'm more OK with using `open_modal` in that it's not user-facing, so if some refactor breaks it, it's basically OK.